packages/uui-button/lib/uui-button.element.ts

```diff
@@ -475,12 +502,27 @@ export class UUIButtonElement extends FormControlMixin(
   }
 
   render() {
-    return html`
-      <button id="button" ?disabled=${this.disabled} aria-label="${this.label}">
-        ${this.renderState()} ${this.renderLabel()}
-        <slot name="extra"></slot>
-      </button>
-    `;
+    return this.href
+      ? html`
+          <a
+            id="button"
+            aria-label=${this.label}
+            href=${ifDefined(!this.disabled ? this.href : undefined)}
+            target=${ifDefined(this.target || undefined)}
+            rel=${ifDefined(this.target === '_blank' ? 'noopener' : undefined)}>
```

Comment:
Do you think we should add a way to overrule this? Say maybe you don't want to add 'noopener' or maybe you also want to add 'noreferrer'?

Also I think checking for target=_blank could be too naïve considering you might want to open a page on your own site while preserving the opener, e.g. for "Save & Preview" you open a new tab where perhaps you would like to keep the reference to the opener tab.

Perhaps we could build a function that validated the location.pathname against the url instead to add the `rel` attribute, or maybe just let it be up to people themselves to add it?

Reply:
Yeah, good one. I have added 'noreferrer' as well.

This was all to ensure that someones secret backoffice doesn't leak itself. We should look into the need to be able to set the rel, but for now I think we should keep it out.

There is a ton of other attributes that we do not enable to set either and I´m not sure we have the right approach/technical solution on how to set these 'native attributes'. I´m thinking about A11Y attributes & events. Its not such a big set for Anchor tag, but for input and other form controls we have a lot of things we cant set jet. And would be good to investigate if we can avoid prop-drilling. :-)